powersimdata/network/usa_tamu/constants/plants.py

```diff
@@ -1,3 +1,18 @@
+__all__ = [
```

Comment:
Is this only meant to be used for loading the `ModelImmutables`? If so I'd consider calling it something custom, to indicate it's not handled by python itself (not sure if there are real risks, just for communicating intent to anyone reading the code).

Reply:
What do you have in mind regarding name: `to_be_listed`, `discoverable`, `searchable`, `attributes`?

Reply:
I didn't have anything in mind, something like `_keys`/`_constants`/`_exports` are what pop into my head, but anything that doesn't overlap with builtin python is good.

Reply:
Done